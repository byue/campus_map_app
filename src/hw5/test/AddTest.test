# Tests that a graph can be created, and 9 nodes
# can be added to the graph. 

# Create a graph
CreateGraph graph1

# Add 9 nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6
AddNode graph1 n7
AddNode graph1 n8
AddNode graph1 n9

# Print the nodes in the graph
ListNodes graph1