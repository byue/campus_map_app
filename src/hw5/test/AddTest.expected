# Tests that a graph can be created, and 9 nodes
# can be added to the graph. 

# Create a graph
created graph graph1

# Add 9 nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1
added node n7 to graph1
added node n8 to graph1
added node n9 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2 n3 n4 n5 n6 n7 n8 n9